helpers/routeHelpers.js

```diff
@@ -43,15 +43,15 @@ export const schema = {
     role: {
         post: Joi.object().keys({
             name: Joi.string().required(),
-            color: Joi.string().required().regex(/^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/)
+            color: Joi.string().required().regex(/^([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/)

```

Comment:
These regex don't work correctly
Need to fix them

Reply:
I just thought it's redundant to include the '#' character since it's in every instance anyway

Reply:
`/#([0-9a-f]{6}|[0-9a-f]{3})/`

Could you try this one?